src/sgame/CombatFeedback.cpp

```diff
@@ -31,7 +31,7 @@ Log::Logger logger("sgame.cf");
  * @brief Notify a client of a hit
  */
 void HitNotify(gentity_t *attacker, gentity_t *victim,
-               Util::optional<Vec3> point_opt, float damage,
+               vec3_t const& point_opt, float damage,
```

Comment:
This loses the optional-ness of point_opt. I imagine you have a compile error on `!point_opt` below.

Reply:
OK so there was a 2nd commit later. I request that broken commits be squashed with their fixes, so that I don't have to waste time commenting on them.

You'd have to have some code to pass nullptr in the missing vector case... maybe it's easier to leave these optionals until glm is added.

Reply:
fixed